### ISSUE TITLE
Missing alt descriptions for slide images

### ISSUE THREAD

**Issue by youngbrioche** [innoq/SCS#48]
In order to make the site accessible for everyone the slide images should have descriptive alt texts.

**Comment by joyheron** [comment]
For slides with a lot of content, image transcripts may be a preferable method to alt texts

See https://seirdy.one/posts/2020/11/23/website-best-practices/#image-transcripts

**Comment by martinweck** [comment]
has been fixed by https://github.com/innoq/SCS/pull/63
